pkg/dms/meta/paths.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package meta
```

Comment:
It's better to reuse the existing `DirPaths` structure instead of re-defining a new one. You can move the exists one into a common package.

Reply:
ok